### PR TITLE
added docs for piptools

### LINKED ISSUE #5: Add pip-tools docs
We are using [pip-tools](https://github.com/nvie/pip-tools) for installing/updating python packages.

### PR DESCRIPTION
**Pull request by mmadil**
fixes #5

### CODE REVIEW

File: README.dev.md [hunk 1]
```diff
@@ -56,15 +57,19 @@ class SomeClass(models.Model):
 
 # Setup
 
-Minimum requirements: **python3, pip & postgres**.
-We advise to use [`virtualenv`][virtualenv] with python3 interprator.
+Minimum requirements: **python3, pip, & postgres**.
```
Comment (akarambir) [inline README.dev.md]: Do we need a comma here? :smile_cat:

File: README.dev.md [hunk 2]
```diff
@@ -56,15 +57,19 @@ class SomeClass(models.Model):
...
+Please go through the following instruction notes in order to get started -
+
+* [Postgres installation instructions][install-postgres]
+* [pip-tools installation and usage instructions](https://github.com/nvie/pip-tools)
 
 ## Getting up and running
 
 ```
-pip install -r requirements.txt
+pip-sync   # for installing depedencies with pip-tools
```
Comment (akarambir) [inline README.dev.md]: People try to run every command in this section, without realizing that they are doing something twice. Maybe we can add regular pip installment in parenthesis or something.

Comment (mmadil) [inline README.dev.md]: Running pip install after pip-sync will not make much of a difference because pip-sync has already installed the dependencies.

**Comment by akarambir** [comment]
Looks good. Thank you

**Comment by akarambir** [comment]
@mmadil Can you please squash your commits into 1 so that I can merge this.

**Pull request merged**